### PR TITLE
Header/Nav 

### LINKED ISSUE #3: Header/Nav
Mobile:

Retirement_Fund___T__Rowe_Price

Desktop:
Retirement_Fund___T__Rowe_Price

### PR DESCRIPTION
**Pull request by bstopp**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #3 

Test URLs:
- Before: https://main--troweprice-rf--hlxsites.hlx.page/
- After: https://feat-nav--troweprice-rf--hlxsites.hlx.page/

### CODE REVIEW

File: blocks/header/header.css [hunk 1]
```diff
@@ -1,275 +1,145 @@
...
+.header.block .nav-sections a {
+  display: block;
+  padding: 14px 24px;
+  font-family: var(--body-font-family);
+  font-size: var(--body-font-size-m);
+  line-height: var(--line-height-m);
+  letter-spacing: var(--letter-spacing-xs);
+  text-decoration: none;
+  background-color: transparent;
+  color: var(--light-blue);
```
Comment (dnbute) [inline blocks/header/header.css:98]: On the original page, while hovering over one of the nav tabs, the colors switch, i.e the background is light blue and the text is white

File: blocks/header/header.js [hunk 2]
```diff
@@ -85,6 +34,54 @@ function toggleMenu(nav, navSections, forceExpanded = null) {
...
 
+/**
+ * Builds the hamburger menu Div.
+ * @returns {HTMLDivElement}
+ */
+function buildBrand() {
+  const brand = document.createElement('div');
+  brand.classList.add('nav-brand');
+  brand.innerHTML = `
+    <img class="logo" src="./styles/images/bright-blue-logo.png" alt="T. Rowe Price Trusty logo">
```
Comment (sergiucoman) [inline blocks/header/header.js:45]: Just wondering, is there a better place for this kind of "content"?

**Review by sergiucoman — APPROVED** [review]
Apart from @dnbute 's comment that it does not look exactly as the original (which I think it is accepted) it looks good to me.

File: blocks/header/header.css (hunk 1 above)
Comment (bstopp) [reply blocks/header/header.css:98]: This is fixed in latest push.

**Review by sergiucoman — APPROVED** [review]

**Pull request merged**